README.md

```diff
@@ -233,3 +318,9 @@ There are two mentions related to Ukraine in the Twiter Algo repo. Whereas [one
 + Admittedly out of date, but still useful, is the [RecSys Wiki](https://www.recsyswiki.com/wiki/Main_Page).
 
 + The latest edition of the [Recommender Systems Handbook](https://link.springer.com/book/10.1007/978-1-0716-2197-4) is also a good book that covers the field well.
+
++ There is this Chinese-language [video tutorial series](https://www.youtube.com/playlist?list=PLvOO0btloRntAi-VnV06M1Bu0X1xljUUP) on recommendation systems. The author was previously an assistant professor and now works at Xiaohongshu (Pinterest equivalent in China).
```

Comment:
we'll probably want to stick to English-language resources here since the doc/target audience is English